app/controllers/concerns/overturned_ico_params.rb

```diff
@@ -1,16 +1,27 @@
 module OverturnedICOParams
   extend ActiveSupport::Concern
 
-  def create_overturned_ico_params
-    params.require(:case_overturned_sar).permit(
-        :original_ico_appeal_id,
-        :original_case_id,
-        :reply_method,
-        :email,
-        :postal_address,
-        :received_date_dd, :received_date_mm, :received_date_yyyy,
-        :external_deadline_dd, :external_deadline_mm, :external_deadline_yyyy,
-        )
+  def create_ico_overturned_foi_params
+    params.require(:case_overturned_foi).permit(
+      :original_ico_appeal_id,
+      :original_case_id,
+      :reply_method,
+      :email,
+      :postal_address,
+      :received_date_dd, :received_date_mm, :received_date_yyyy,
+      :external_deadline_dd, :external_deadline_mm, :external_deadline_yyyy,
+    )
   end
 
+  def create_ico_overturned_sar_params
```

Comment:
do we need to be case type specific with the method names? They seem to use the same params.

Reply:
No. This must be left-over from when we thought they'd be more separate. I'll look at merging these.

Reply:
Ah, I see now. The issue is that one requires `case_overturned_sar` and the other `case_overturned_foi` as the type of case params (see line 17). I'll extract the param list to one place but I think that's the best we can reasonably do.